### PR TITLE
DataAggregator Change Log

### PR DESCRIPTION
**Pull request by JoaoSE-Skyline**
Add Data Aggregator DxM RNs for 3.0.X releases

### CODE REVIEW

**Comment by JoaoSE-Skyline** [comment]
@wimme @jaklke @PedroDebevere

**Review by PedroDebevere — APPROVED** [review]

File: release-notes/DxMs/DataAggregator_change_log.md [hunk 1]
```diff
@@ -4,10 +4,57 @@ uid: DataAggregator_change_log
...
+To ensure better security, access to Data Aggregator's web interface now requires the user to authenticate itself.
+
+##### API Gateway module registration [ID_38570]
+
+Data Aggregator will now register itself with API Gateway allowing for an overview of all node instances.
+
+##### Extend Data Aggregator to have a CRUD API for Data Sources [ID_37309]
+
+Data Aggregator API has been extended from the ability control the Jobs flow (start, stop and check status) to a full CRUD.
+This has been done for both Jobs and Data Sources, allowing for configuration with less manual steps.
```
Comment (MariekeGO) [inline release-notes/DxMs/DataAggregator_change_log.md:52]: I find this description a little confusing. Could you rephrase this keeping the end user in mind? E.g. what is "control the Jobs flow to a full CRUD"? I'm also confused by the first sentence mentioning "the Jobs flow" and the second one saying that this "has been done for both Jobs and Data Sources".

**Review by MariekeGO — CHANGES_REQUESTED** [review]

File: release-notes/DxMs/DataAggregator_change_log.md [hunk 2]
```diff
@@ -4,10 +4,57 @@ uid: DataAggregator_change_log
...
+Python environment and libraries come with Data Aggregator's installer so no need to configure that after installation.
+Making it possible to start using the feature out of the box.
+
+In addition, several Python libraries come pre-installed but more can be installed following [these steps](xref:Data_Sources_Setup#installing-extra-python-packages).
+
+##### Enforce authentication when accessing Data Aggregator DxM web interface via reverse proxy [ID_38275]
+
+To ensure better security, access to Data Aggregator's web interface now requires the user to authenticate itself.
+
+##### API Gateway module registration [ID_38570]
```
[the remark this answers is not in the document]
Comment (JoaoSE-Skyline) [reply release-notes/DxMs/DataAggregator_change_log.md:45]: The RN has an "oversimplified" and slightly incomplete explanation of what was changed.
The purpose of that RN was to implement the registration procedure in DataAggregator and also make it more robust since we had encountered issues without the retry mechanism on Data API.
Part of its functionality was also implemented in [38234](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm.aspx?ID=38234)

File: release-notes/DxMs/DataAggregator_change_log.md [hunk 3]
```diff
@@ -4,10 +4,57 @@ uid: DataAggregator_change_log
...
+
+The installer has been extended to include the bs4 pip package (and its dependencies) by default in the DataAggregator installer.
+
+#### 07 February 2024 - Enhancement - DataAggregator 3.0.1 - Improve DataAggregator Migrator [ID_38682]
+
+Improvements have been made to the [migrator tool](xref:Data_Aggregator_Migrator) to better allow converting GQI queries from the [previous syntax to the new one](xref:Data_Aggregator_settings#gqi-queries).
+
+This includes:
+- Extra information added to the Help command (_-h_ or _--help_)
+- Change how the [DataAPI soft launch](xref:Overview_of_Soft_Launch_Options#dataapi) verification is applied in order to facilitate the migration process for users yet using it.
```
[the remark this answers is not in the document]
Comment (MariekeGO) [reply release-notes/DxMs/DataAggregator_change_log.md:26]: This is purely a grammar and spelling change.

[the remark this answers is not in the document]
Comment (JoaoSE-Skyline) [reply release-notes/DxMs/DataAggregator_change_log.md]: Bad interpretation on my side

File: release-notes/DxMs/DataAggregator_change_log.md (hunk 1 above)
Comment (JoaoSE-Skyline) [reply release-notes/DxMs/DataAggregator_change_log.md]: Slight rephrasing

**Review by MariekeGO — APPROVED** [review]
